book/src/framework/components/chainlink/nodeset.md

```diff
@@ -42,6 +42,8 @@ This component requires some Blockchain to be deployed, add this to config
 Then configure NodeSet
 ```toml
 [nodeset]
+  # unique NodeSet name
+  name = "don"
```

Comment:
curious: we do not validate whether it is indeed unique?

Reply:
also, maybe mention that its optional if there's only one nodeset? unless it's not optional ;-)

Reply:
It is not, we are going the path of least amusement for now and let these things be explicit. If further configuration becomes ugly we refactor it later.